tests/sentry/incidents/endpoints/test_project_alert_rule_index.py

```diff
@@ -343,9 +306,115 @@ def test_async_lookup_outside_transaction(self, mock_uuid4, mock_get_channel_id)
         with self.feature("organizations:incidents"), self.tasks():
             resp = self.get_response(self.organization.slug, self.project.slug, **test_params)
         assert resp.status_code == 400
-        assert (
-            mock_get_channel_id.call_count == 3
-        )  # Did not increment from the last assertion because we early out on the validation error
+        # Did not increment from the last assertion because we early out on the validation error
+        assert mock_get_channel_id.call_count == 3
+
+    @responses.activate
+    def test_success_response_from_sentry_app(self):
```

Comment:
Tests that endpoint properly handles metric alerts with Sentry App actions (POST)